tests/kernel/profiling/profiling_api/prj.conf

```diff
@@ -8,8 +8,9 @@ CONFIG_TEST_HW_STACK_PROTECTION=n
 # to check idle thread
 CONFIG_SYS_POWER_MANAGEMENT=y
 CONFIG_SYS_POWER_LOW_POWER_STATES=y
-CONFIG_IDLE_STACK_SIZE=512
+CONFIG_SYS_PM_POLICY_APP=y
+CONFIG_IDLE_STACK_SIZE=768
```

Comment:
why idle stack size is increased?

Reply:
The example uses hook called from idle thread to print some info. After changes the hook is called one-level deeper and there was no space for the printk().